lang/en/tool_heartbeat.php

```diff
@@ -43,12 +43,16 @@
 $string['configauthmissing'] = 'Configured auth methods are not currently enabled.';
 $string['setinitialauthstate'] = 'Initial auth state for heartbeat auth check set.';
 $string['authcorrect'] = 'Auth methods correctly configured.';
+$string['builtinallowediplist'] = 'Builtin IP Blocking Configuration';
+$string['builtinallowediplist_desc'] = 'A second allowed IP list which makes it easier to have some IPs forced in config.php and others editable in the UI';
```

Comment:
```suggestion
$string['builtinallowediplist_desc'] = 'This allowed IP list would allow some IPs to be editable in the UI in addition to those forced in config.php';
```

Reply:
Done